src/open_inwoner/components/templatetags/button_tags.py

```diff
@@ -62,6 +62,7 @@ def button(text, **kwargs):
         - aria_label: string | The HTML aria-label attribute if different from the text.
         - extra_classes: string | Extra classes that need to be added to the button
         - extra_attributes: dict | Extra attributes that need to be added to the button
+        - bold
```

Comment:
A comment would be nice
```suggestion
        - bold: bool | if the button text is bold
```

Reply:
oh no's - I should have deleted this, we're not using this.